core/drivers/crypto/hisilicon/include/hpre_main.h

```diff
@@ -0,0 +1,104 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023 HiSilicon Limited.
+ */
+#ifndef _HPRE_MAIN_H
+#define _HPRE_MAIN_H
+
+#include "hisi_qm.h"
+#include "rng_support.h"
+
+/*
+ * Version number maintenance rule:
+ * first.second.third: year.month.change num
+ */
+#define HPRE_MODULE_VERSION "22.11.2"
```

Comment:
We have git for that. It's not a big deal to keep this define, if it's the only one. But it's bound to raise questions from people outside your team since it appears you're maintaining a more updated out-of-tree driver. Questions like: This looks old, is this the latest version of the driver? Are the latest security fixes applied? And so on.

Reply:
> We have git for that. It's not a big deal to keep this define, if it's the only one. But it's bound to raise questions from people outside your team since it appears you're maintaining a more updated out-of-tree driver. Questions like: This looks old, is this the latest version of the driver? Are the latest security fixes applied? And so on.

According to your suggestion, we delete the information of the version.